app/code/Magento/WishlistGraphQl/Model/Resolver/UpdateProductsInWishlist.php

```diff
@@ -160,4 +186,76 @@ private function getWishlist(?int $wishlistId, ?int $customerId): Wishlist
 
         return $wishlist;
     }
+
+    /**
+     * Update wishlist Item and set data from request
+     *
+     * @param int $itemId
+     * @param DataObject $buyRequest
+     * @param wishlistFactory $wishlist
+     *
+     * @return WishlistOutput
+     * @throws GraphQlInputException
+     * @throws \Magento\Framework\Exception\AlreadyExistsException
+     */
+    private function updateItem($itemId, $buyRequest, $wishlist)
+    {
+        $item = $wishlist->getItem((int)$itemId);
+
+        if (!$item) {
+            throw new GraphQlInputException(__('We can\'t specify a wish list item.'));
+        }
+
+        $product = $item->getProduct();
+        $productId = $product->getId();
+
+        if ($productId) {
+            $buyRequest->setData('action', 'updateItem');
+            $product->setWishlistStoreId($item->getStoreId());
+            $cartCandidates = $product->getTypeInstance()->processConfiguration($buyRequest, clone $product);
+
+            /**
+             * If the product with options existed or not
+             */
+            if (is_string($cartCandidates)) {
+                throw new GraphQlInputException(__('The product with options does not exist.'));
+            }
+
+            /**
+             * If prepare process return one object
+             */
+            if (!is_array($cartCandidates)) {
+                $cartCandidates = [$cartCandidates];
+            }
+
+            foreach ($cartCandidates as $candidate) {
+                if ($candidate->getParentProductId()) {
+                    continue;
+                }
+                $candidate->setWishlistStoreId($item->getStoreId());
+                $qty = $buyRequest->getData('qty') ? $buyRequest->getData('qty') : 1;
+                $item->setOptions($candidate->getCustomOptions());
+                $item->setQty($qty)->save();
```

Comment:
Please, do not use deprecated methods in your code. The `::save` method of `AbstractModel` is deprecated since 2.1